{"name": "test_add", "status": "failed", "statusDetails": {"message": "AssertionError: condition fail aish\nassert (1 + 1) == 3", "trace": "self = <Drivers.locators_ty.alerts_handeling_assignments.assignments_testing.test_file1.TestExample object at 0x0000029372FDFB50>\n\n    def test_add(self):\n>       assert 1+1==3, 'condition fail aish'\nE       AssertionError: condition fail aish\nE       assert (1 + 1) == 3\n\ntest_file1.py:4: AssertionError"}, "start": 1618415632967, "stop": 1618415632967, "uuid": "7ee8b6ff-858d-4d3f-b8d1-c3296a1027c6", "historyId": "f9be7b95d733c0192da39647fd3bc964", "testCaseId": "cf3d00025486649dd25b63aa566814ed", "fullName": "test_file1.TestExample#test_add", "labels": [{"name": "suite", "value": "test_file1"}, {"name": "subSuite", "value": "TestExample"}, {"name": "host", "value": "DESKTOP-2QKIU49"}, {"name": "thread", "value": "18324-MainThread"}, {"name": "framework", "value": "pytest"}, {"name": "language", "value": "cpython3"}, {"name": "package", "value": "test_file1"}]}